extensions/opentelemetry/opentelemetry-exporter-jaeger/runtime/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/runtime/JaegerExporterRuntimeConfig.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.opentelemetry.exporter.jaeger.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.opentelemetry.runtime.config.OtelConnectionConfig;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "otel.exporter.jaeger")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface JaegerExporterRuntimeConfig extends OtelConnectionConfig {
+
+    /**
+     * Sets the Jaeger base endpoint to connect to. If unset, defaults to {@value Constants#DEFAULT_JAEGER_ENDPOINT}.
+     */
+    @Override
+    //    @WithConverter(TrimmedStringConverter.class)
+    Optional<String> endpoint();
+
+    /**
+     * Jaeger traces exporter configuration
+     */
+    JaegerExporterTracesConfig traces();
+    // TODO metrics();
+    // TODO logs();
+
+    class Constants {
+        public static final String DEFAULT_JAEGER_ENDPOINT = "http://localhost:14250";
```

Comment:
Should that depend on Jaeger supporting OTLP natively (since 1.35?)

Reply:
I guess this is Jaeger-legacy and folks would otherwise use the OTLP-exporter

Reply:
I imagine so. This comes from the spec: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#jaeger-exporter